app/konnect/dev-portal/index.md

```diff
@@ -40,8 +40,24 @@ For developers:
 
 * [Generate credentials for an application](/konnect/dev-portal/applications/dev-gen-creds/) - This guide walks you through the process of generating and deleting API keys on the Dev Portal platform.
 
+### Contextual developer analytics 
+
+The Developer Portal supports built-in analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This feature gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable **Dashboard**. Each application has it's own dashboard that provides a high-level summary of **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
```

Comment:
```suggestion
You can view analytics for developer applications from within the {{site.konnect_short_name}} Dev Portal. This gives developers the ability to gain insights into all of their applications and the relationships between applications and APIs using a fully customizable dashboard. 

Each application has its own dashboard, which provides a high-level summary of the **Number of Requests**, **Average Error Rate**, and **p99 Latency**, and charts for the following data points: 
```
Rephrased some repetitive wording.

 "Each application has its own dashboard that provides" doesn't work - Either:
* "Each application has a dashboard that provides"
* "Each application has its own dashboard, which provides"

I suggested the "which" phrasing, but you're welcome to go with the other option.